src/app/(post-login)/submission/components/ProgramMenu.tsx

```diff
@@ -16,63 +16,99 @@
  * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
  * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-'use client';
+"use client";
 
-import Loader from '@/app/components/Loader';
-import { notNull } from '@/global/utils/types';
-import { css } from '@/lib/emotion';
-import { useQuery } from '@apollo/client';
-import { MenuItem } from '@icgc-argo/uikit';
-import Link from 'next/link';
-import { notFound, usePathname } from 'next/navigation';
-import { MouseEventHandler, useState } from 'react';
-import SIDEMENU_PROGRAMS from '../gql/SIDEMENU_PROGRAMS';
+import Loader from "@/app/components/Loader";
+import SIDEMENU_PROGRAMS from "@/app/gql/SIDEMENU_PROGRAMS";
+import { useSubmissionSystemStatus } from "@/app/hooks/useSubmissionSystemStatus";
+import { notNull } from "@/global/utils/types";
+import { css } from "@/lib/emotion";
+import { useQuery } from "@apollo/client";
+import { MenuItem } from "@icgc-argo/uikit";
+import Link from "next/link";
+import { notFound, usePathname } from "next/navigation";
+import { MouseEventHandler, useState } from "react";
+
+const StatusMenuItem = () => {
+  const { isDisabled: isSubmissionSystemDisabled } =
+    useSubmissionSystemStatus();
+  return (
+    <div
+      css={css`
+        display: flex;
+        justify-content: space-between;
+        width: 100%;
+        align-items: center;
+        padding-right: 15px;
+      `}
+    >
+      Register Samples
+      {/* {isSubmissionSystemDisabled ? (
+        <Icon name="lock" fill="accent3_dark" width="15px" />
+      ) : clinicalRegistrationHasError ? (
+        <Icon name="exclamation" fill="error" width="15px" />
+      ) : clinicalRegistrationInProgress ? (
+        <Icon name="ellipses" fill="warning" width="15px" />
+      ) : null} */}
```

Comment:
Is this block related to something in progress?

Reply:
It's related to a piece of work that I could easily scope out of this ticket as it's already a very large ticket with lots of code change.